lib/linguist/languages.yml

```diff
@@ -8077,6 +8069,19 @@ wisp:
   - ".wisp"
   tm_scope: source.clojure
   language_id: 420
+Workflow Description Language:
+  aliases:
+  - wdl
+  type: programming
+  color: "#42f1f4"
+  extensions:
+  - ".wdl"
+  interpreters:
+  - miniwdl
+  - Cromwell
```

Comment:
Are these really shell interpreters, ie commonly used in shebang lines? From a quick search for the `.wdl` extension, I can't find either of these in shebang lines.

```suggestion
```

Reply:
My understanding is that they are interpreters, but shebangs are almost never used in workflow languages. In fact, strictly speaking, the WDL spec seems to imply they should not be used, [as the first line of the file should instead be used to declare what version of WDL is being used](https://github.com/openwdl/wdl/blob/main/versions/1.0/SPEC.md#versioning).

In other words, you can do this, but no one ever does:
```
#! miniwdl run

version 1.0 

[insert valid WDL 1.0 here]
```

Reply:
Yeah, this is a bit unclear in the comments. Neither linguist nor GitHub will be using these to actually interpret the files. These are used to help the shebang strategy when identifying the language [here](https://github.com/github/linguist/blob/9358f090deb76511b613ab024b704c1c0f13a4c2/lib/linguist/shebang.rb#L20) so extensionless files, common to interpreted languages like shell scripts, can be correctly identified.

Reply:
> In other words, you can do this, but no one ever does:
>
>     #! miniwdl run

Slightly off-topic, but a portable interpreter directive should use [`env(1)`](https://man.openbsd.org/env.1) to resolve the interpreter (in this case, `miniwdl`). As for the version-on-first-line requirement… well, a lot of languages mandate similar constructs, but make exceptions for hashbangs and/or modelines. So I wouldn't take the implied limitation too literally.

Reply:
Since miniwdl is installed by pip and Cromwell is a jar file, I'm actually not quite sure how to make them more portable.

Reply:
I don't think it really matters in the context of this PR. These interpreters are not used in shebang lines so these don't belong in the `languages.yml` file for this language. Please remove them as requested earlier.